addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -4,64 +4,148 @@ This binding integrates the possibility to execute arbitrary shell commands.
 
 ## Supported Things
 
-Currently, the binding supports a single type of Thing, being the ```command``` Thing.
+Currently, the binding supports a single type of Thing, being the `command` Thing.
 
 ## Binding Configuration
 
 The binding does not require any specific configuration.
 
-Note that the commands are executed in the context and with the privileges of the process running the java virtual machine.
+
+**Linux:**
+Note that the commands are executed in the context and with the privileges of the process running the Java Virtual Machine.
+On a Linux system the system user `openhab` needs to have the privileges needed to execute your intended command.
+It is advised to test the correct operation of the command in the scope of the `openhab` user on the command line first:
+
+```shell
+sudo -u openhab <YOUR COMMAND>
+```
 It is not advised to run the virtual machine as superuser/root.
 
 ## Thing Configuration
 
-The `command` Thing requires the command to execute on the shell.
+The "command" thing requires the command to execute on the shell.
 Optionally one can specify:
 
--   a transformation to apply on the execution result,
--   an interval, in seconds, the command will be repeatedly executed,
--   a time-out, in seconds, the execution of the command will time out, and lastly,
--   a boolean parameter to make the command execute immediately every time the state of the input channel has changed.
+- `transform` - A transformation to apply on the execution result,
```

Comment:
Please link transformation to: https://docs.openhab.org/addons/transformations.html